src/test/kotlin/graphics/scenery/tests/examples/advanced/AsyncTextureExample.kt

```diff
@@ -0,0 +1,171 @@
+package graphics.scenery.tests.examples.advanced
+
+import graphics.scenery.*
+import org.joml.Vector3f
+import graphics.scenery.backends.Renderer
+import graphics.scenery.attribute.material.Material
+import graphics.scenery.compute.ComputeMetadata
+import graphics.scenery.compute.InvocationType
+import graphics.scenery.primitives.Plane
+import graphics.scenery.textures.Texture
+import graphics.scenery.textures.UpdatableTexture
+import graphics.scenery.utils.Image
+import graphics.scenery.utils.RingBuffer
+import graphics.scenery.volumes.Volume
+import net.imglib2.type.numeric.integer.UnsignedByteType
+import org.joml.Vector3i
+import org.lwjgl.system.MemoryUtil
+import java.io.File
+import kotlin.concurrent.thread
+import kotlin.system.measureTimeMillis
+import kotlin.time.Duration.Companion.nanoseconds
+
+/**
+ * Example loading a large texture asynchronously
+ *
+ * @author Ulrik Günther <hello@ulrik.is>
+ */
+class AsyncTextureExample: SceneryBase("Async Texture example", 512, 512) {
```

Comment:
This needs to be updated with the new version we made that actually shows the texture changes.

Reply:
Done in 0e4e09a